File: fizzbuzz.cpp
#include <iostream>
using namespace std;

int main(){

    int intFizz,intBuzz, intUpTo;

    cin>>intFizz>>intBuzz>>intUpTo;

    for(int i =1; i<=intUpTo;i++){
        if(i%intFizz==0 && i%intBuzz==0)
            cout<<"FizzBuzz\n";
        else if(i%intFizz==0)
            cout<<"Fizz\n";
        else if(i%intBuzz==0)
            cout<<"Buzz\n";
        else
            cout<<i<<endl;

    }

return 0;}

File: fizzbuzz.py
Fizz=int(input())
Buzz=int(input())
N=int(input())

for i in range(1,N+1):
    if(i%Fizz==0 and i%Buzz==0):
        print("FizzBuzz")
    elif(i%Fizz==0):
        print("Fizz")
    elif(i%Buzz==0):
        print("Buzz")
    else:
        print(i)
